pkg/config/config.go

```diff
@@ -13,12 +13,16 @@ const (
 	CILIUM_EBPF_ENGINE = "cilium"
 )
 
+const (
+	SNIFFER_CONFIG = "SNIFFER_CONFIG"
```

Comment:
`SNIFFER_CONFIG_ENV_VAR` might be a more meaningful name.

Also, please combine const declarations